packages/cli/src/commands/validator/status.ts

```diff
@@ -0,0 +1,117 @@
+import { Address } from '@celo/contractkit'
+import { flags } from '@oclif/command'
+import { cli } from 'cli-ux'
+import { range } from 'lodash'
```

Comment:
Not sure if `lodash` is part of cli `package.json`....
And personally, i would avoid it :P